masterlist.yaml

```
@@ -8057,6 +8057,45 @@ plugins:
       - crc: 0xA082C1A4
         util: 'TES4Edit v4.0.3'
 
+  - name: 'Rathunas(01|-MidasMagic|-BetterMusicSystem)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/45880/'
+        name: 'Rathunas'
+  - name: 'Rathunas01.esp'
```

Needs a few load after rules, just in case they set `MidasSpells.esp` to load after UL.

```suggestion
  - name: 'Rathunas01.esp'
    after:
      - 'MidasSpells.esp'
      - 'Rathunas-MidasMagic.esp'
```